digdag-docs/src/command_reference.rst

```diff
@@ -367,21 +367,27 @@ In the config file, following parameters are available
 * database.maximumPoolSize (integer, default: available CPU cores * 32)
 * database.leakDetectionThreshold (HikariCP leakDetectionThreshold milliseconds in integer. default: 0. To enable, set to >= 2000.)
 * database.migrate (enable DB migration. default: true)
-* archive.type (type of project archiving, "db" or "s3". default: "db")
+* archive.type (type of project archiving, "db", "s3" or "gcs". default: "db")
 * archive.s3.endpoint (string. default: "s3.amazonaws.com")
 * archive.s3.bucket (string)
 * archive.s3.path (string)
 * archive.s3.credentials.access-key-id (string. default: instance profile)
 * archive.s3.credentials.secret-access-key (string. default: instance profile)
 * archive.s3.path-style-access (boolean. default: false)
-* log-server.type (type of log storage, "local" , "null", or "s3". default: "null". This parameter will be overwritten with "local" if ``-O, --task-log DIR`` is set.)
+* archive.gcs.bucket (string)
+* archive.gcs.credentials.json.path (string. if not set, auth with local authentication information.)
+* archive.gcs.credentials.json.content (string. if not set, auth with local authentication information.)
```

Comment:
In my understand, if both path and content are not set , local authentication information is used, right ?
and if both is set, path has priority, correct ?

Reply:
@yoyama 
Yes, your understanding is correct.
Please check if the wording is good.

Reply:
Please add these explanation to document.